test/test_smoothing.py

```diff
@@ -1,72 +0,0 @@
-from firedrake import *
-from movement import *
-import numpy as np
```

Comment:
Are we totally discarding the test for the laplacian smoothing method?

Reply:
The demos get run as tests that are very similar to these tests anyway. That's why I'm dropping the test versions - to avoid duplication.